pkg/api/validation/validation.go

```diff
@@ -1934,6 +1935,16 @@ func ValidateSeccompPodAnnotations(annotations map[string]string, fldPath *field
 	return allErrs
 }
 
+func validateAppArmorProfile(p string, fldPath *field.Path) field.ErrorList {
+	if p == "docker/default" {
```

Comment:
In the proposal it's called "runtime/default" which I would prefer to avoid problems with rkt and others. We have had the same discussion for seccomp (/cc @pweil-).